cli/lib/kontena/cli/services/containers_command.rb

```diff
@@ -18,7 +18,7 @@ def execute
         puts "  rev: #{container['deploy_rev']}"
         puts "  node: #{container['node']['name']}"
         puts "  dns: #{container['name']}.#{current_grid}.kontena.local"
-        puts "  ip: #{container['overlay_cidr'].split('/')[0]}"
+        puts "  ip: #{container['overlay_cidr'].nil? ? '' : container['overlay_cidr'].split('/')[0]}"
```

Comment:
This can be simplified a little bit by casting value always to string: `container['overlay_cidr'].to_s.split('/')[0]`